bundles/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/DSCAlarmActiveBinding.java

```diff
@@ -951,43 +914,35 @@ private void verboseTroubleStatusHandler(EventObject event) {
             itemName = getItemName(dscAlarmItemTypes[i], 0, 0);
 
             if (itemName != "") {
```

Comment:
I know you didn't add this line, but comparing a String with an operator will only work accidentally. Could you change it (and any other occurrences) to something like if (!("".equals(itemName)) or if (StringUtils.isNotEmpty(itemName))?